versatile-api/src/main/java/com/fb/versatile_api/HotelController.java

```diff
@@ -0,0 +1,70 @@
+package com.fb.versatile_api;
+
+import java.util.List;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HotelController {
+
+    private Hotel hotel = new Hotel();
+
+    @GetMapping("/hotel")
+    public String getHotelInfo() {
+        return String.format(
+            "Hotel Name: %s.<br>" +
+                "Hotel Address: %s.<br>" +
+                "Contact: %s.<br><br>" +
+                "Total rooms: %s<br>",
+            hotel.getHotelName(),
+            hotel.getHotelAddress(),
+            hotel.getTelephoneNumber(),
+            hotel.roomCount()
+        );
+    }
+
+    @GetMapping("/hotel/listrooms")
```

Comment:
As a recommended practice, we don't use keywords in `urls` such as `list`, 'get', 'create',  'add', 'delete', 'update', 'modify', etc. Your service 'Url' and used  `http` methods should be self-explanatory.

Here are some good read around this concept

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods
https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design